### ISSUE TITLE
support for aria-braille* properties

### ISSUE THREAD

**Issue by pkra** [nvaccess/nvda#11829]
The ARIA Editors' Draft proposes the aria-braillelabel and aria-brailleroledescription property (as part of ARIA 1.3), see also https://w3c.github.io/aria/#aria-braillelabel, https://w3c.github.io/aria/#aria-brailleroledescription.

Would NVDA consider support for these properties?

**Comment by pkra** [comment]
Ping @michaelDCurran and @feerrenrut who were at the AIM workshop in 2018 where the idea started out.

Somewhat related: https://github.com/nvaccess/nvda/pull/8340 (even though the input property is still not covered by the ARIA Editors' draft).

**Comment by leonardder** [comment]
Implementing `brailleroledescription` should be not that difficult. As far as I know, there is no such thing as a braille name in NVDA now, but that could be pretty helpful as well.

**Comment by pkra** [comment]
Thank you for the positive comment, @leonardder. If there are any other communication channels for NVDA where I should post this, please let me know.

**Comment by pkra** [comment]
Just a quick follow up.

The new aria braille properties are still on track for aria 1.3. There's now support in Gecko, WebKit and Chromium. Orca also has support. It would be great to get support in NVDA.

See also the overview issue https://github.com/w3c/aria/issues/1331 for more links to various bits and pieces.

**Comment by CoBC** [comment]
I also approved this proposal. I find this aria peoperties very helpful to shorten some names in Braille for example.

**Comment by BeatriceFeltre** [comment]
Hello,
I've seen this issue, and I'm also interested in the aria-braillelabel property. is there any release prediction of property management?
Thank you so much
Beatrice

**Comment by BeatriceFeltre** [comment]
> Hello, I've seen this issue, and I'm also interested in the aria-braillelabel property. is there any release prediction of property management? Thank you so much Beatrice

Hello, is there any news about it?
thank you 
Beatrice

**Comment by dmca-glasgow** [comment]
I would just like to add a strong use case for `aria-braillelabel`.  I am working on making university maths coursework more accessible.  Building on top of projects like MathJax and Speech Rule Engine, I can generate a speech and braille version of the maths.  Complex maths formulae generate an extremely complex speech string, while the Nemeth or Euro braille equivalent is orders of magnitude simpler and more concise.  Using `aria-label` and `aria-braillelabel` on each maths element, I can straightforwardly provide both options to the end user.  

I hope this feature can be added to NVDA soon.  Thanks.

**Comment by b-g-h** [comment]
In order for NVDA users to be able to use Mathjax formulas both audibly as formula descriptions and in written form with Eurobraille code (Pseudo-LaTeX) on the braille display , it would be very helpful if aria-braillelabel support could also be implemented in NVDA. 
In addition to language, our students need a written representation in order to actively engage in mathematics. 
Thank you very much.
